dss-pades/src/main/java/eu/europa/esig/dss/pdf/pdfbox/PdfBoxSignatureService.java

```diff
@@ -277,7 +276,7 @@ public void saveDocumentIncrementally(PAdESSignatureParameters parameters, Outpu
 	}
 
 	protected COSName getSubFilter() {
-		return PDSignature.SUBFILTER_ETSI_CADES_DETACHED;
+		return null;
```

Comment:
Leave the default sub filter value empty to enable the use of whatever the signature parameters have configured.